jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java

```diff
@@ -118,6 +118,29 @@ public void callback(final ProcessInstanceSummary process) {
                     default:
                         break;
                 }
+                view.getStateText().setText(statusStr);
+                
+                String slaComplianceStr = constants.Unknown();
```

Comment:
Would it be work moving this logic into a separated method that could also be easily tested?